habitat-lab/habitat/datasets/rearrange/configs/hab/rearrange.yaml

```diff
@@ -134,7 +134,7 @@ object_samplers:
     params:
       object_sets: ["hab2"]
       receptacle_sets: ["clutter"]
-      num_samples: [30, 30]
+      num_samples: [30, 40]
```

Comment:
This change enables the speedup despite more overall objects being sampled by providing room to correct unstable placements instead of failing.
I can roll it back before merge if necessary for backwards compatibility.